google/cloud/documentai_toolbox/utilities/utilities.py

```diff
@@ -24,8 +24,8 @@
 from google.cloud.documentai_toolbox.wrappers.document import _get_storage_client
 
 
-def print_gcs_document_tree(gcs_bucket_name: str, gcs_prefix: str) -> None:
-    r"""Prints a tree of filenames in Cloud Storage folder.
```

Comment:
I think it makes sense to return a list of documents, but I also think that there should be a separate function for printing (and not have the list function print it). Just seems like each function should only have one purpose

Maybe the print function could call `list_gcs_document_tree` in the background, then just print the list.